Source/Scene/BoxEmitter.js

```diff
@@ -0,0 +1,45 @@
+/*global define*/
+define([
+        '../Core/defaultValue',
+        '../Core/Cartesian3',
+        '../Core/Math',
+        './Particle'
+    ], function(
+        defaultValue,
+        Cartesian3,
+        CesiumMath,
+        Particle) {
+    "use strict";
+
+    var BoxEmitter = function(options) {
```

Comment:
I've changed all of the constructors, I think I copied the "var" syntax from one of the enum classes.

The BoxEmitter just emits particles from a box with width, height and depth parameters.  I don't have a particular use case for a box emitter vs  the other emitter types, but it's apparently useful b/c it's implemented in Unity as a shape type :smile:.  

Playing around in unity you can generate some neat effects by setting one or more of the dimensions of the box to 0 so you could generate particles from a flat plane or a line.